### PR TITLE
Windows

### PR DESCRIPTION
**Pull request by strues**
- add prop-types as a dependency to prevent react-hot-loader from breaking build (see #414)
- add cross-env for starting tasks which break on Windows.
- add FAQ for when running develop and javascript files aren't loading even though the build is successful.
modify the clean task to execute a function rather than use the ${npm bin} exec command, which doesn't work on Windows.
- disable eslint rule for linebreak-style to prevent overload of errors when working on a Windows machine. 

If the user has git setup properly for Windows with  `git config --global core.autocrlf input` the linebreak-style rule wont matter. When committing the code to a remote repository, the line ending will convert from CRLF to LF if any, but when checking out from the remote directory, no conversion from LF to CRLF is performed.

### CODE REVIEW

**Comment by ctrlplusb** [comment]
Fantastic @strues

File: package.json [hunk 1]
```diff
@@ -61,13 +61,15 @@
     "app-root-dir": "1.0.2",
     "colors": "1.1.2",
     "compression": "1.6.2",
+    "cross-env": "^4.0.0",
```
Comment (birkir) [inline package.json]: Exact version, there is a handy `-E` argument for yarn 👍

File: package.json [hunk 2]
```diff
@@ -61,13 +61,15 @@
...
     "dotenv": "4.0.0",
     "express": "4.15.2",
     "helmet": "3.5.0",
     "hpp": "0.2.1",
     "modernizr": "3.4.0",
     "normalize.css": "6.0.0",
     "offline-plugin": "4.6.2",
+    "prop-types": "^15.5.4",
```
Comment (birkir) [inline package.json]: Same for this one.

**Comment by ctrlplusb** [comment]
Looks like your misfortune is gonna benefit others @strues 😊

Keen to get this through, however, are we gonna have to wait for the 15.5 shockwave to subside?

**Comment by ctrlplusb** [comment]
@strues now that we have v15.5 in master I think we all good to get this merged in too?

File: .eslintrc [hunk 3]
```diff
@@ -14,6 +14,10 @@
     // A jsx extension is not required for files containing jsx
     "react/jsx-filename-extension": 0,
     // This rule struggles with flow and class properties
-    "react/sort-comp": 0
+    "react/sort-comp": 0,
+    // ignore linebreak style. the CRLF / LF endings wont matter
+    // if a windows user correctly converts CRLF to LF upon commits otherwise
+    // there are errors every line.
+    "linebreak-style": 0
```
Comment (ctrlplusb) [inline .eslintrc]: Such a goodie to be aware of.  I am surprised AirBNB doesn't define this as the default.

Comment (strues) [inline .eslintrc]: Windows needs to conform to the standards of the other two superior platforms :)

**Comment by ctrlplusb** [comment]
Ready to get this merged in! :)

@strues do you mind resolving conflicts with latest master and getting setting the dependencies to exact?  After this we can merge 👍

**Comment by strues** [comment]
@ctrlplusb taking care of it now

**Comment by ctrlplusb** [comment]
Rad thanks

**Comment by ctrlplusb** [comment]
Good spot. Great to have so many eyes over the project. Awesome guys.

**Comment by strues** [comment]
nice catch. Looks like it was left out after i rebased.

**Comment by ctrlplusb** [comment]
Sweet! Thanks @strues - awesome to have this in 👌

**Comment by birkir** [comment]
Great job guys, you may feel pretty good going into this weekend.

**Comment by diondirza** [comment]
great job @strues

**Pull request merged**